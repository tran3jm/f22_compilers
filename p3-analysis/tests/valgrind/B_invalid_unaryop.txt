==2962169== Memcheck, a memory error detector
==2962169== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==2962169== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==2962169== Command: ../decaf inputs/invalid_unaryop.decaf
==2962169== 
Type mismatch: int is incompatible with bool on line 4
==2962169== 
==2962169== HEAP SUMMARY:
==2962169==     in use at exit: 0 bytes in 0 blocks
==2962169==   total heap usage: 1,818 allocs, 1,818 frees, 221,524 bytes allocated
==2962169== 
==2962169== All heap blocks were freed -- no leaks are possible
==2962169== 
==2962169== For lists of detected and suppressed errors, rerun with: -s
==2962169== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
