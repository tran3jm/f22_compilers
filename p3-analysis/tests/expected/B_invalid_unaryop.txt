Type mismatch: int is incompatible with bool on line 4
